tests/e2e/features/presentationViewer.feature

```diff
@@ -10,4 +10,5 @@ Feature: markdown presentation viewer
 
   Scenario: preview markdown file in presentation viewer
     When user "admin" previews a markdown file "test-markdown.md" in presentation viewer
```

Comment:
```suggestion
    When user "admin" previews the markdown file "test-markdown.md" in presentation viewer
```

or 

```suggestion
    When user "admin" previews markdown file "test-markdown.md" in presentation viewer
```

I think the specific article "the" (or no article) is used in English in this situation where a specific file is mentioned... alternatively you could say 

```suggestion
    When user "admin" previews a markdown file with the name "test-markdown.md" in presentation viewer
```

but that's quite a long sentence...